java/pom.xml

```diff
@@ -24,14 +24,106 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>org.apache.arrow</groupId>
                 <artifactId>arrow-plasma</artifactId>
                 <version>0.10.0-SNAPSHOT</version>
             </dependency>
+
```

Comment:
This is great! I was also planing to sanitize dependency versions across sub modules. Glad that you already did this.